Sources/SwiftDocC/Infrastructure/ConvertActionConverter.swift

```diff
@@ -146,6 +163,8 @@ package enum ConvertActionConverter {
         // Wait for any concurrent updates to complete.
         resultsGroup.wait()
         
+        signposter.endInterval("Render", renderSignpostHandle)
```

Comment:
Should we add a sign post for the following code about emitting the metadata/digest files?

Reply:
Sure. I intended for us to add more signpost intervals overtime but I can add a few more in this initial batch.

Added in https://github.com/swiftlang/swift-docc/pull/1112/commits/40bb242e8a70914f9139158b9204681f411d08ee